src/System.Net.Http/src/System/Net/Http/Managed/ValueStringBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text
+{
+    internal struct ValueStringBuilder
```

Comment:
I guess the nearest thing is `List<T>` but the code is a little more convoluted, it has an extra `version` field, and there is no efficient way to get a string out of it...